packages/components/app/styles/components/side-nav/main.scss

```diff
@@ -66,56 +72,100 @@
 
 .hds-side-nav__menu-toggle-button {
   position: absolute;
+  top: -2px;
+  left: calc(var(--hds-app-sidenav-border-width) * -1);
   z-index: 1;
+  display: flex;
+  box-sizing: content-box;
+  padding: calc(var(--token-side-nav-toggle-button-padding-vertical) - var(--hds-app-sidenav-border-width)) var(--token-side-nav-toggle-button-padding-horizontal);
   color: var(--token-color-foreground-high-contrast);
+  text-align: right;
```

Comment:
what is this for?

Reply:
leftover, removed in 531c5b78cad742c6695961175c3f5afcd62334e3